lib/puppet/parser/functions/master_classifer_settings.rb

```diff
@@ -1,5 +1,5 @@
 module Puppet::Parser::Functions
-  newfunction(:master_classifer_settings, :type => :rvalue) do |args|
+  newfunction(:master_classifer_settings, type: :rvalue) do
```

Comment:
this doesn't work.  Will switch back to :type => :rvalue and disable the lint check for now.